src/amo/components/ServerHtml/index.js

```diff
@@ -124,14 +124,11 @@ export default class ServerHtml extends Component {
   }
 
   getFontPreload() {
-    const { assets, htmlLang } = this.props;
-    // Preload relevant minimal subset font if available for this language.
+    const { assets } = this.props;
+    // Preload variable font(s) with `preload` in their name.
```

Comment:
This comment should probably be updated.

Reply:
That's why I get from mixing multiple branches into one :)

Reply:
Fixed in 5466c2ca359ec909f8c5bba57b7e7390adb33749 - logic was ok, the comment was just outdated